ui/src/app/utils/dates.tsx

```diff
@@ -0,0 +1,25 @@
+import {isBlank} from './index';
+
+export const MILLIS_PER_DAY = 24 * 60 * 60 * 1000;
+export const daysFromNow = (timeInMillis: number): number => Math.floor((timeInMillis - Date.now()) / MILLIS_PER_DAY);
+
+// To convert datetime strings into human-readable dates
+export function displayDate(time: Number): string {
+  const date = new Date(Number(time));
```

Comment:
Existing issue: `Number(time)` is interesting; seems like it should be unnecessary given the type. If it is needed for some reason, a comment would be good.

Reply:
removed and tested